### PR TITLE
adds uniform_zoom to ImageDataGenerator and random_zoom

### PR DESCRIPTION
**Pull request by julien-ur**
### Summary

This modification gives the user the ability to define, if the zoom transformation should be the same on the xy axes or seperate for each axis.

Hope this is the right way to suggest such change. Sorry if not, but I never did this before :)

### Related Issues
The parameter has to be introduced in the main keras.preprocessing library as well.

### PR Overview

- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [ ] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by Dref360** [comment]
What's the use case for this? I don't see the added value of such a feature.

**Comment by fchollet** [comment]
Closing outdated PR. Note that the Keras Preprocessing repository is no longer in use, instead please just use the dataset utilities provided in `tf.keras.utils` ([see docs](https://keras.io/api/data_loading/)).
